examples/persistent_launch.ts

```diff
@@ -0,0 +1,20 @@
+// Import Astral
+import { launch } from "../mod.ts";
+
+import { resolve } from "https://deno.land/std@0.205.0/path/resolve.ts";
+
+// Launch the browser
+const browser = await launch({
+  // Setting the directory where the browser will store user-specific data, such as cookies, local storage, and other browsing data.
+  userDataDir: resolve("./my-browser-data"),
```

Comment:
Do we need resolve here?

Reply:
If you do not do this, the directory will be created in the folder where the browser is launched.

Reply:
We should probably automatically resolve it (in `launch`). I feel like making it spawn where the browser is launched would definitely not be behavior people would expect.